src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Apprentice/Details.cshtml

```diff
@@ -6,14 +6,39 @@
     ViewData["Title"] = "Apprentice Details";
 }
 
-
 <h1 class="govuk-heading-xl">@Model.ApprenticeName</h1>
 
-
-@*https://localhost:5001/10005077/apprentices/4MBBNM*@
-
 <div class="govuk-grid-row govuk-!-margin-bottom-2">
     <div class="govuk-grid-column-two-thirds">
+
+        @if (Model.Status == ApprenticeshipStatus.Stopped ||
+            Model.Status == ApprenticeshipStatus.Live ||
+            Model.Status == ApprenticeshipStatus.Paused ||
+            Model.Status == ApprenticeshipStatus.WaitingToStart)
+        {
+            <details id="" class="govuk-details" data-module="govuk-details">
+                <summary class="govuk-details__summary">
+                    <span class="govuk-details__summary-text">
+                        Help with changes in apprentice's circumstances
+                    </span>
+                </summary>
+                <div class="govuk-details__text">
+                    <ul class="govuk-list govuk-list--bullet">
```

Comment:
The first p tag needs to be left open so that the bulleted list is inside it. The ul tag needs to be moved inside the p tag so that the li tags are the immediate children.

```
                <div class="govuk-details__text">
                    
                        <p class="govuk-body">If an apprentice you're training needs to move to a new employer, you should:
                        
                            <ul class="govuk-list govuk-list--bullet">        
                                <li>contact their original employer and ask them to stop the apprentice record</li>
                                <li>speak to the new employer to agree new training dates, price and request permission to move the apprentice to their apprenticeship service account</li>
                            </ul>

                        </p>

                        <p class="govuk-body">Once you've agreed these details, you can change the employer.</p>

                        <p class="govuk-body">Non-levy employers will not need apprenticeship funding reserved when taking on an apprentice from another employer.</p>
                    
                </div>
```

Also, can we give the details tag an id so that the automation test can find it.

Reply:
Changed to the following:
`
            <details id="change-of-employer-details" class="govuk-details" data-module="govuk-details">
                <summary class="govuk-details__summary">
                    <span class="govuk-details__summary-text">
                        Help with changes in apprentice's circumstances
                    </span>
                </summary>
                <div class="govuk-details__text">
                    
                    <p class="govuk-body">If an apprentice you're training needs to move to a new employer, you should:

                        <ul class="govuk-list govuk-list--bullet">
                            <li>contact their original employer and ask them to stop the apprentice record</li>
                            <li>speak to the new employer to agree new training dates, price and request permission to move the apprentice to their apprenticeship service account</li>
                        </ul>

                        Once you've agreed these details, you can change the employer.<br /><br />

                        Non-levy employers will not need apprenticeship funding reserved when taking on an apprentice from another employer.
                    </p>
                </div>
            </details>
`